microhapdb/cli/marker.py

```diff
@@ -92,7 +92,10 @@ def display(
             for locus in loci.values():
                 print(locus.fasta)
         elif view_format == "offsets":
-            table = pd.concat([marker.definition for marker in markers])
+            loci = defaultdict(Locus)
+            for marker in markers:
+                loci[marker.locus].markers.append(marker)
+            table = pd.concat([locus.definition for locus in loci.values()])
```

Comment:
In the CLI, group markers by locus and use `locus.definition` rather than individual `marker.definition` attributes.